docs/voldoen-aan-wetten-en-regels/hulpmiddelen/standaarden.md

```diff
@@ -0,0 +1,144 @@
+---
+hide: navigation
+---
+
+## Wat zijn standaarden?
+Standaarden zijn afspraken die worden vastgelegd om ervoor te zorgen dat producten, processen of diensten voldoen aan specifieke eisen op het gebied van kwaliteit, veiligheid en prestaties. Ze worden ontwikkeld door standaardisatieorganisaties, meestal op initiatief van belanghebbenden die een behoefte aan een bepaalde standaard zien, zoals de Europese Commissie, bij of na het ontwikkelen van Europese regelgeving.
+
+Standaarden zijn er op allerlei gebieden. Momenteel speelt de ontwikkeling van standaarden voor de Europese [AI-verordening](../ai-verordening.md) een belangrijke rol voor de betrouwbare ontwikkeling en gebruik van AI. Vandaar dat deze pagina focust op _standaarden voor AI_.
+
+Er zijn verschillende soorten standaarden, zoals normen, Technische Rapporten (TR) en Technische Specificaties (TS).
```

Comment:
Misschien is het beter om dit weg te laten, want anders is het wel veel. Het belangrijkste is dat organisaties begrijpen dat er een verschil is tussen 'gewone' standaarden en geharmoniseerde normen.

Reply:
TR/TS/norm is volgens mij minder belangrijk. Het belangrijkste onderscheid is met de *geharmoniseerde* normen, want die zijn als enige toegesneden op de AIA en bieden de presumption of conformity en daarmee juridische zekerheid.

Maar je kunt het wat mij betreft opnemen hoor! Als het de aandacht maar niet afleidt van wat het allerbelangrijkste is: de geharmoniseerde normen.

Reply:
Goed bezig jullie! Dit hele stuk is een heel mooie toevoeging!